lib/cmd/common/admin-policies.js

```diff
@@ -165,32 +170,38 @@ function update(guid, policyId, type, config, checkUserExists, checkUserApproved
     "checkUserExists" : checkUserExists || false,
     "checkUserApproved" : checkUserApproved || false
   };
-
-  common.doApiCall(fhreq.getFeedHenryUrl(), "/box/srv/1.1/admin/authpolicy/update", payload, "Error updating policy: ", function(err, data){
+  var url = fhreq.getFeedHenryUrl();
+  var msg = i18n._("Error updating policy: ");
+  var cb = function(err, data) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "cb" as this name is already used in declaration at line 152. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2814)